packages/app/src/components/Sidebar/Bookmarks.tsx

```diff
@@ -1,16 +1,80 @@
-import React, { FC } from 'react';
 
+import React from 'react';
+
+import { DevidedPagePath } from '@growi/core';
+import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
+import { UncontrolledTooltip } from 'reactstrap';
 
-const Bookmarks: FC = () => {
-  const { t } = useTranslation('');
+import LinkedPagePath from '~/models/linked-page-path';
+import { useSWRInifiniteBookmarkedPage } from '~/stores/bookmark';
+import { useCurrentUser } from '~/stores/context';
+
+import PagePathHierarchicalLink from '../PagePathHierarchicalLink';
+
+import InfiniteScroll from './InfiniteScroll';
+
+
+const BookmarksItem = ({ data }) => {
```

Comment:
should be declared Props type like other components
```suggestion
const BookmarksItem = (props: Props) => {
```